### PR TITLE
try to fix OSS CI error

### PR DESCRIPTION
**Pull request by tiandiao123**
Differential Revision: D43259005

### CODE REVIEW

**Review by weiwangmeta — APPROVED** [review]
LGTM

File: third_party/xnnpack.buck.bzl [hunk 1]
```diff
@@ -86,7 +86,17 @@ def define_xnnpack(third_party, labels = [], XNNPACK_WINDOWS_AVX512F_ENABLED = F
     fb_xplat_cxx_library(
         name = "operators",
         # srcs have to include HOT_SRCS to be able to build on ARVR
```
Comment (malfet) [inline third_party/xnnpack.buck.bzl:88]: Please remove this comment if this is no longer the case

Comment (tiandiao123) [reply third_party/xnnpack.buck.bzl:88]: Ok!

Comment (weiwangmeta) [reply third_party/xnnpack.buck.bzl:88]: Suggest merging now to catch nightly. Follow up to address this comment issue?

Comment (tiandiao123) [reply third_party/xnnpack.buck.bzl:88]: Ok! yeah!

**Review by digantdesai — APPROVED** [review]
LGTM

File: third_party/xnnpack.buck.bzl (hunk 1 above)
Comment (malfet) [reply third_party/xnnpack.buck.bzl:88]: Hmm, but it did not catch the nightly, so perhaps it should be changed now?

**Pull request closed without merging**